plugins/DevicesDetection/lang/de.json

```diff
@@ -51,7 +51,7 @@
         "GenericDevice": "Allgemeines %s",
         "MobileDevice": "Mobilgerät",
         "Tablet": "Tablet",
-        "Peripheral": "Dezentral",
+        "Peripheral": "Peripheriegeräte",
```

Comment:
@sgiehl If you have a better translation for this new category, I'm happy to use it instead

Reply:
Can't think of a better one right now... 🤔